readme.md

```diff
@@ -0,0 +1,104 @@
+# Qwant Maps
+
+## Architecture
+
+Qwant Maps can be seen as 4 separated components:
+
+* a tile server
+* a search engine (geocoder)
+* a front end
+* an API to detail Pois
+
+### Tile server
+
+The tile server is a combination of 2 great opensource projects:
+
+* [kartotherian](https://github.com/kartotherian/kartotherian), the wikimedia stack to have a highly available tile server. It is itself based on a number of [mapbox](https://www.mapbox.com/) components.
+* [OpenMapTiles](https://github.com/openmaptiles/openmaptiles), for their great and flexible tile schema.
+
+OpenMapTiles makes it possible for Qwant Maps to have a [easy to define/extend](https://github.com/QwantResearch/openmaptiles) vector tile schema.
+
+All the data (see [below](#tilesdata)) are imported in a [Postgresql](https://www.postgresql.org/) database using mainly [imposm](https://imposm.org/docs/imposm3/latest/). All the world's vector tiles are then generated using kartotherian's [tilerator](https://github.com/kartotherian/tilerator) and stored in [cassandra](http://cassandra.apache.org/). This enable Qwant Maps to have a fast and scalable way to serve tiles.
+
+### Geocoder
+
+Qwant Maps uses as its geographical search engine (called a geocoder) [Mimirsbrunn](https://github.com/CanalTP/mimirsbrunn).
```

Comment:
Suggestion for better formulation:
The geographical search engine (also called geocoder) used for Qwant Maps is ...